_posts/2018-12-26-til-how-to-run-ecto-migrations-in-production.md

```diff
@@ -0,0 +1,55 @@
+---
+author: Kate Travers
+author_link: https://github.com/ktravers
+categories: til
+date: 2018-12-16
+layout: post
+title: How to Run Ecto Migrations on Production
```

Comment:
Let's include "TIL" in the title. Maybe: "TIL: How to Run Ecto Migrations on Production"